src/network_modelling/network.py

```diff
@@ -55,7 +56,7 @@ def __init__(self, prm):
 
             # ieee network and corresponding incidence matrix
             self.net = pandapower.networks.ieee_european_lv_asymmetric('on_peak_566')
-            self.n_non_flex_homes = len(self.net.asymmetric_load) - self.n_homes
+            self.n_non_flex_homes = self.n_homesP
```

Comment:
how to you allocate a location to the passive home? are you randomly selecting nodes on the network? do you then model the whole network or only the buses with active home + the buses with passive homes?

Reply:
Currently, I assign them in the order of the loads in the network. For example, if we have 3 flexible homes, they will be assigned to the first three homes of the actual network, i.e. bus 34, 47, 70. Then, if we also have 3 passive homes, they will be assigned to the next three loads in the network, e.g. bus 73, 74 and 83. The original IEEE network had defined where exactly the 55 loads are and I simply reuse that.
Eventually, I will assign them randomly throughout the network. It will just make the experiments less comparable. I will probably do this in #32, if that's okay with you.

Reply:
I model the whole network because I think that modelling only a part might be tricky since we don't have a logical order of buses and lines. For example, sometimes line 1 links line 2 but sometimes line 80 is only surrounded by line 75 and 83. Same with buses.

Reply:
> I model the whole network because I think that modelling only a part might be tricky since we don't have a logical order of buses and lines. For example, sometimes line 1 links line 2 but sometimes line 80 is only surrounded by line 75 and 83. Same with buses.

that is good!

Reply:
> Currently, I assign them in the order of the loads in the network. For example, if we have 3 flexible homes, they will be assigned to the first three homes of the actual network, i.e. bus 34, 47, 70. Then, if we also have 3 passive homes, they will be assigned to the next three loads in the network, e.g. bus 73, 74 and 83. The original IEEE network had defined where exactly the 55 loads are and I simply reuse that.
> Eventually, I will assign them randomly throughout the network. It will just make the experiments less comparable. I will probably do this in #32, if that's okay with you.

but when we use the "first x busses" are they located close to the source bus or are the "first x buses" randomly scatted around? (though I understand this is not you setting where there are, this is just the order in which they are set by the IEEE input network)

Reply:
ok thanks for clarifying, and could you please make sure this is seeded so that running a same run twice should give you exactly the same thing for reproducibility (All the random generators should already be seeded so that should already be the case but please do check, thank you!)